packages/atomic/src/components/atomic-modal/atomic-modal.pcss

```diff
@@ -0,0 +1,88 @@
+@import '../../global/global.pcss';
+
+[part='backdrop'] {
+  pointer-events: none;
+
+  @screen desktop-only {
+    display: grid;
+    place-items: center;
+    grid-template-areas:
+      '. .     .'
+      '. modal .'
+      '. .     .';
+    grid-template-columns: 1fr auto 1fr;
+    grid-template-rows: 1fr auto 3fr;
+
+    transition: background-color 500ms ease-in-out;
+    &.opened {
+      background-color: rgba(40, 40, 40, 0.8);
+    }
+  }
+
+  &.opened {
+    pointer-events: auto;
+  }
+}
+
+atomic-focus-trap {
+  display: contents;
+}
+
+[part='container'] {
+  grid-area: modal;
+  overflow: hidden;
+
+  @screen mobile-only {
```

Comment:
For now, my default assumption is that the mobile look of smart snippet should look almost exactly like the refine modal, but that can easily be changed. I'll tag Johnny on it in the next PR.